extensions/circuit/src/base_alu/mod.rs

```diff
@@ -6,6 +6,6 @@ pub use core::*;
 
 pub type WomBaseAluChip<F> = VmChipWrapperWom<
     F,
-    WomBaseAluAdapterChip<F>,
+    WomBaseAluAdapterChip<F, 2, 4, 4>,
```

Comment:
does it make sense to use `RV32_REGISTER_NUM_LIMBS` instead of 4?
maybe then also an RV64_REGISTER_NUM_LIMBS

Reply:
done